src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -184,9 +194,9 @@ export const PositionsTable = ({ positions, displayLine = false }: PositionsProp
     }
 
     if (userRoleMetadata.role === BORROWER_POSITION_ROLE) {
-      if (position.status === 'CLOSED') return [];
+      if (position.status === CLOSED_STATUS) return [];
 
-      if (position.status === 'PROPOSED') {
+      if (position.status === PROPOSED_STATUS) {
```

Comment:
I think u still need to check who is the `maker` on the proposal soyou know who needs to confirm the proposed position

Reply:
for borrower + lender logic

Reply:
this was actually dead code so I deleted it. see other comments about deadcode in `getUserPositionActions`